build.gradle

```
@@ -186,3 +186,26 @@ subprojects {
         archives javadocJar
     }
 }
+
+if (!org.gradle.internal.os.OperatingSystem.current().isWindows()) {
+    task checkCopyright(type: Exec) {
+        description = 'Run tools/check-copyright.sh'
+        group = 'Verification'
+
+        commandLine 'tools/check-copyright.sh'
+    }
+
+    task ktlint(type: Exec) {
```

Ideally, we would make this a `JavaExec` task that runs `ktlint` directly (which would then [work on Windows](https://github.com/shyiko/ktlint#usage), too), and also use the `JavaExec` task's `standardOutput` property to put output into a `ByteArrayOutputStream` which we can then filter in the same way as `ktlint.sh` does, and then remove `ktlint.sh` altogether.

---

Something like this maybe:

```suggestion
task ktlint(type: JavaExec) {
    description = 'Runs ktlint.'
    group = 'Verification'

    classpath = files('tools/ktlint')
    main = 'com.github.shyiko.ktlint.Main'
    ignoreExitValue = true
    standardOutput = new ByteArrayOutputStream()

    doLast {
        def output = standardOutput.toString().trim().tokenize('\n')
        output.removeAll { it ==~ /.*(Wildcard import|Unexpected spacing before).*/ }
        if (!output.empty) {
            throw new GradleException("Please fix the following issues:\n${output.join('\n')}")
        }
    }
}
```